sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/schema/BeamTableUtils.java

```diff
@@ -151,10 +151,16 @@ public static Object autoCastField(Schema.Field field, Object rawObj) {
         case INT32:
           return Integer.valueOf(raw);
         case INT64:
+          if (raw.equals("")) {
+            return null;
```

Comment:
Just curious about this. Is this because `autoCastField` is how CSV is converted to INT64? I happen to be looking at other things and found this.

Reply:
Yes, data in format of CSV files are read by this autoCastField method and converted to various types.

When input data field is empty, it needs to be treated as NULL manually, otherwise it will cause error.